app/editor/src/features/admin/series/SeriesMerge.tsx

```diff
@@ -0,0 +1,126 @@
+import { FormikForm } from 'components/formik';
+import React from 'react';
+import { useLocation, useParams } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import { useLookup } from 'store/hooks';
+import { useSeries } from 'store/hooks/admin';
+import {
+  Button,
+  ButtonVariant,
+  Col,
+  FieldSize,
+  FormikSelect,
+  FormikText,
+  IOptionItem,
+  Modal,
+  OptionItem,
+  Row,
+  useModal,
+} from 'tno-core';
+
+import { defaultSeries } from './constants';
+import { ISeriesForm } from './interfaces';
+import * as styled from './styled';
+import { toForm } from './utils';
+
+/** The page used to view and edit series the administrative section. */
+const SeriesMerge: React.FC = () => {
+  const { id } = useParams();
+  const [, api] = useSeries();
+  const { state } = useLocation();
+  const { toggle, isShowing } = useModal();
+  const [{ series }] = useLookup();
+
+  const [targetSeries, setTargetSeries] = React.useState<ISeriesForm>(
+    (state as any)?.series ?? defaultSeries,
+  );
+  const [seriesOptions, setSeriesOptions] = React.useState<IOptionItem[]>([]);
+  const [mergeSeriesSourceOption, setMergeSeriesSourceOption] = React.useState<IOptionItem>();
+
+  const targetSeriesId = Number(id);
+
+  React.useEffect(() => {
+    if (!!targetSeriesId && targetSeries?.id !== targetSeriesId) {
+      setTargetSeries({ ...defaultSeries, id: targetSeriesId }); // Do this to stop double fetch.
```

Comment:
Make sure the double fetch isn't the idiotic think React does in development mode...

Reply:
in all honesty, I just copied that boilerplate code from the original `SeriesForm` page